metaflow/plugins/argo/argo_live_run.py

```diff
@@ -88,9 +88,7 @@ def trigger(
 
         # convert dicts/lists to json-valid strings
         for key in parameters:
```

Comment:
(minor and not part of this PR) `for key, value in parameters.items()` slightly simplify the code.